httpclient/src/main/java/io/opentracing/contrib/httpcomponents/SpanExec.java

```diff
@@ -32,25 +32,25 @@ public CloseableHttpResponse execute(HttpRoute route, HttpRequestWrapper request
         Tracer.SpanBuilder spanBuilder = tracer.buildSpan(String.format("HTTP %s", request.getMethod()))
                 .asChildOf(contextSpan.get())
                 .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_CLIENT);
-        try(Span span = spanBuilder.start()) {
-            this.tracer.inject(span.context(), Format.Builtin.HTTP_HEADERS, new HttpRequestTextMap(request));
-            return contextSpan.set(span).<CloseableHttpResponse, HttpException, IOException>supplyException2(() -> {
-                HttpTagger tagger = taggerFactory.create(span, context);
-                tagger.tagRequest(request.getOriginal());
-                tagger.tagTarget(route.getTargetHost());
-                final CloseableHttpResponse response;
-                try {
-                    response = exec.execute(route, request, context, execAware);
-                } catch (HttpException | IOException e) {
-                    Tags.ERROR.set(span, true);
-                    throw e;
-                } finally {
-                    tagger.tagContext();
-                }
-                tagger.tagResponse(response);
-                return response;
-            });
-        }
+                
```

Comment:
is this newline needed?

Reply:
It's not, but stylistically, I prefer to have a space between a multi-line series of function calls and the next line, just because I think it makes it easier to read. I don't have strong opinions on it, though.